charts/prometheus-alerts/README.md

```diff
@@ -20,6 +19,19 @@ those changes in the `charts/simple-app`, `charts/daemonset-app` and
 
 ## Upgrade Notes
 
+### 1.4.x -> 1.5.x
+
+**BREAKING: The AlertSelectorValidity alert rules requires kube-state-metrics.**
+
+An alert was introduced in that requires kube-state-metrics to be installed in the cluster. If
```

Comment:
All of these metrics rely on kube-state-metrics ... this is not a change from the previous behavior.